### PR TITLE
upgrade tidbit and token-price-oracles deps

### PR DESCRIPTION
**Pull request by mikec**

### CODE REVIEW

File: contracts/Oracles/MedianPriceOracle.sol [hunk 1]
```diff
@@ -79,15 +79,13 @@ contract MedianPriceOracle is ScalarPriceOracleBase, TimedOracle {
...
+    (, uint endDate) = medianDataFeed.resultByIndex(endIndex);
 
     bool endIndexWithinTimeframe = endDate <= resolutionDate;
     bool nextIndexAfterTimeframe = true;
 
-    if (medianDataFeed.doesIndexExistFor(endIndex.add(1))) {
-      (, uint nextDate) = medianDataFeed.resultByIndexFor(endIndex.add(1));
-      nextIndexAfterTimeframe = nextDate > resolutionDate;
-    }
+    (, uint nextDate) = medianDataFeed.resultByIndex(endIndex.add(1));
```
Comment (emaG3m) [inline contracts/Oracles/MedianPriceOracle.sol]: what if the `endIndex` is the last date recorded? Then `medianDataFeed.resultByIndexFor(endIndex.add(1));` would be 0, causing this to revert

Comment (emaG3m) [reply contracts/Oracles/MedianPriceOracle.sol]: I would've thought a test would have failed bc of this...did it not??

Comment (emaG3m) [reply contracts/Oracles/MedianPriceOracle.sol]: https://github.com/levelkdev/futarchy-app/blob/master/test/Oracles/MedianPriceOracle.test.js#L69

Comment (mikec) [reply contracts/Oracles/MedianPriceOracle.sol]: oops good catch! I ran the test suite but somehow missed the failing test

**Comment by mikec** [comment]
@emaG3m I added that check back

**Comment by emaG3m** [comment]
👍

**Pull request merged**